compiler/luci-interpreter/include/luci_interpreter/import/GraphBuilderRegistry.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
I think it would be better to revise all `2021` to `2022`.
Happy new year!

Reply:
Happy New Year! =)

Reply:
Replaced with 2022 year!